docs/docs/api/Tasks/fetchTaskByDay.md

```diff
@@ -53,10 +45,28 @@ Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE1OTU4MjQyN
 
 #### Failure
 
+```json
```

Comment:
The immediate failures object that gets returned should be the right structure as what's truly returned from the API. What I included as a guideline was just as an example. 

Our failures look something like 
![image](https://user-images.githubusercontent.com/24628243/161081154-1c8e679b-f086-40e5-93f8-ca0ab6426964.png)

as opposed to
![image](https://user-images.githubusercontent.com/24628243/161081205-3b118112-b748-4374-bac4-37baf183d376.png)

As noted in quite a few PR re-reviews, please make the changes accordingly on the Fetch Single Task` file as well as the `Fetch Task By Day` file